### PR TITLE
Add oppia@0.0.3

### PR DESCRIPTION
**Pull request by kevinlee12**
Checklist for **Pull request** or **lib adding request issue** follows the conventions.

NB: the source of the added file is https://github.com/oppia/oppia/blob/develop/assets/scripts/oppia-player-0.0.3.min.js

# Profile of the lib
 * Git repository (required): https://github.com/oppia/oppia/
 * GitHub / Bitbucket popularity (required):
   - Count of watchers: 111
   - Count of stars: 378
   - Count of forks: 469
# Essential checklist
 * [ ] I'm the author of this library
   * [ ] I would like to add link to the page of this library on CDNJS on website / readme
 * [ ] This lib was not found on cdnjs repo
 * [x] No already exist / duplicated issue and PR
 * [x] The lib has notable popularity
   * [x] More than 200 [Stars / Watchers / Forks] on [GitHub / Bitbucket]
   * [ ] More than 800 downloads stats per month on npm registry
 * [x] Project has public repository on famous online hosting platform (or been hosted on npm)

# Auto-update checklist
 * [ ] Has valid tags for each versions (for git auto-update)
 * [ ] Auto-update setup
 * [ ] Auto-update target/source is valid.
 * [ ] Auto-update filemap is correct.

# Git commit checklist
 * [x] The first line of commit message is less then 50 chars, be clean and clear, easy to understand.
 * [ ] The parent of the commit(s) in the PR is not old than 3 days.
 * [x] Pull request is sending from a non-master branch with meaningful name.
 * [x] Separate unrelated changes into different commits.
 * [x] Use rebase to squash/fixup dummy/unnecessary commits into only one commit.
 * [ ] Close corresponding issue in commit message
 * [ ] Mention related issue(s), people in commit message, comment.

### CODE REVIEW

**Comment by kevinlee12** [comment]
Hi @extend1994, I performed the git operations using local commands. As for the PR updated homepage and license as suggested and squashed the v0.0.3 commit with it as well. Thanks!

**Comment by kevinlee12** [comment]
Hi @extend1994, I separated the commits and placed the meta updates before the library updates. Thanks!

File: ajax/libs/oppia/package.json [hunk 1]
```diff
@@ -1,7 +1,7 @@
 {
   "name": "oppia",
   "filename": "oppia-player.min.js",
-  "version": "0.0.2",
+  "version": "0.0.3",
```
Comment (maruilian11) [inline ajax/libs/oppia/package.json]: Please move this diff into 05045db. Thanks.

**Comment by kevinlee12** [comment]
Hi @maruilian11, made the requested changes, thanks!

**Comment by extend1994** [comment]
@kevinlee12 Could you please add oppia-player.js v0.0.3 in the commit `Add v0.0.3 for oppia` and mention that you get the files from where? Thank you :)

**Comment by kevinlee12** [comment]
Hi @extend1994, since we didn't add it for the previous versions before, I'm not really inclined to add the non-minified versions for the past versions. I'm happy with the library as-is. Thanks!

**Comment by PeterDaveHello** [comment]
@sufuf3 @extend1994 @maruilian11 when do we drop the requirement on auto-update config?

**Comment by maruilian11** [comment]
@PeterDaveHello Because seems like `oppia-player` is just a embed script for oppia project. And I didn't find `oppia-player`'s own repo or npm package.  https://github.com/oppia/oppia/commit/f732735015eaae24c4c4349c761a122970fd72c2

**Comment by kevinlee12** [comment]
That is correct, the script is only an embed script. As @maruilian11 mentioned, we don't have a separate repo for this script or a npm package.

**Comment by PeterDaveHello** [comment]
We don't add new library that without auto-update, it's a requirement ...

**Comment by kevinlee12** [comment]
Hi @PeterDaveHello, we don't use npm nor version our embedding script the same way that we version our main codebase, so the auto-update systems won't be compatible. Neither are these embedding scripts updated often. Will it be possible for you to reconsider? Thanks!

**Comment by extend1994** [comment]
@PeterDaveHello It's not a new lib. We have hosted old versions of this lib (https://cdnjs.com/libraries/oppia). Would it be okay to add new version like this pull request?

**Comment by PeterDaveHello** [comment]
Sorry, I misunderstand myself, what I wanted to say was our reviewer should ask to have that, but I didn't see it, it'll be better that if we can have auto-update enabled on this on in future, thanks.

**Comment by PeterDaveHello** [comment]
Thanks all!

**Pull request merged**